src/components/menu/menu.svelte

```diff
@@ -222,9 +222,7 @@
 
     border: 1px solid var(--leo-color-divider-subtle);
     border-radius: var(--leo-radius-m);
-    width: max-content;
     min-width: var(--leo-menu-control-width);
-    max-width: var(--leo-menu-popup-max-width);
```

Comment:
Again, this feels slightly off to me. Is there a reason we can't control the width of the slotted content?